cypress/support/utils/testlog.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+function printResult(result) {
+    console.log(`${now()} - Test ${result.state}...`);
+}
+
+function printStep(description) {
+    console.log(`${now()} ====>> ${description}...`);
+}
+
+function printDoubleLine(length) {
+    console.log(`${now} ${"*".repeat(length)}`);
+}
+
+function testHeader(testName) {
+    console.log(`${now()} ${"*".repeat(60)}`);
+    console.log(`${now()} <== ${testName} ==>`);
+    
+}
+
+function reportError(err) {
+    console.log(`${now()} - ${"-".repeat(30)}>`)
+    console.log(`${now()} - ${err.stack} `);
+    console.log(`${now()} - <${"-".repeat(30)}`)
+}
+
+function now() {
+    return Cypress.moment().format("MMMDD-HH:mm.ss");
+}
+
+export { now, printDoubleLine, printResult, testHeader, reportError, printStep }
```

Comment:
Keep consistent naming.  This file is should be a logger, and re-use same javascript vocabulary as `log` instead of `print`.  In the web printing refers to printing on paper or to a device so I would prefer to keep the prefix `log`.

Also, instead of `reportError` I would prefer to use the builtin `console.error` or `console.warn`  I think they already print stack traces